app/api/connectors/wordpress.js

```diff
@@ -58,7 +58,7 @@ export default class WordPressApi {
   }
 
   async getPosts() {
-    const RESPONSE = await this.axios.$get(`wp/v2/posts?_embed`);
+    const RESPONSE = await this.axios.$get(`wp/v2/posts?_embed&per_page=99`);
```

Comment:
To nie rozwiązuje problemu + nawet 50 postów mocno obciąży pierwsze ładowanie strony (vide KIWK)